From: Simon McVittie <smcv@debian.org>
Date: Fri, 8 Oct 2010 08:27:20 +0100
Subject: [PATCH] Version-check libtool correctly in out-of-tree builds

---
 configure |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure b/configure
index 67787b4..ba61f1c 100755
--- a/configure
+++ b/configure
@@ -11076,7 +11076,7 @@ fi
 
 if test "$versioned" = "yes"; then
   # it seems that GNU ld versions since 2.10 are not broken
-  xres=`grep '^VERSION=' libtool | cut -d = -f 2 | cut -d \" -f 2`
+  xres=`grep '^VERSION=' ${srcdir}/ltmain.sh | cut -d = -f 2 | cut -d \" -f 2`
   major=`echo $xres | cut -d . -f 1`
   minor=`echo $xres | cut -d . -f 2`
   pass=0
-- 
